analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/helper_methods/CommentHelperMethods.java

```diff
@@ -1,12 +1,15 @@
-package com.infosupport.ldoc.analyzerj.helperMethods;
+package com.infosupport.ldoc.analyzerj.helper_methods;
 
 import com.github.javaparser.ast.comments.JavadocComment;
 
 import java.util.LinkedHashMap;
 import java.util.Map;
 
 public class CommentHelperMethods {
-  public static String extractSummary(JavadocComment commentText) {
+  private CommentHelperMethods() {
+    // Private constructor to prevent instantiation
+  }
+    public static String extractSummary(JavadocComment commentText) {
```

Comment:
Whitespace seems a bit odd on this line.

Reply:
it's just preference so if anyone looks at it later on to not wonder what it is for.